bind/utils_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bind
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestGetGoVersion(t *testing.T) {
+	var versionTests = []struct {
```

Comment:
@sbinet 
Thank you for letting me know the better way.
I learn a lot from you.